docs/api.md

```diff
@@ -28,11 +28,17 @@ Instead, it *returns* a new, connected component class, for you to use.
      - `value` *(Any)*: Data to passthrough directly to `PromiseState` as an alternative to providing a URL. This is an advanced option used for static data and data transformations. Also consider setting `meta`.
      - `meta` *(Object)*: Metadata to passthrough directly to `PromiseState`. Keys `request`, `response`, and future keys may be overwritten.
 
-Requests specified as functions are not fetched immediately when props are received, but rather bound to the props and injected into the component to be called at a later time in response to user actions. Functions should be pure and return the same format as `mapPropsToRequestsToProps` itself. If a function maps a request to the same name as an existing prop, the prop will be overwritten. This is commonly used for taking some action that updates an existing `PromiseState`. Consider setting `refreshing: true` in such it situation.
+  The following keys may also be defined on an individual request (see their description below at `connect.defaults()`):
```

Comment:
yeah, i figured that was the case. i just pulled it in for now and will think about it. maybe how it is is fine.